app/views/layouts/_labs_main.erb

```diff
@@ -122,7 +122,7 @@
                     <nav>
                         <ul class="pull-right list-inline">
                             <li><a href="/terms-of-use">Terms of use</a></li>
```

Comment:
We can probably get rid of this link - as we don't have any terms of use. Probably replace with a link to the About page